charts/timed/values.yaml

```diff
@@ -307,13 +305,19 @@ redmine:
   url: redmine.example.com
   # redmine.apiKey -- Redmine API Key
   apiKey: ""
-  # redmine.htaccessUser -- Redmine htaccess user
-  htaccessUser: ""
-  # redmine.htaccessPassword -- Redmine htaccess password
```

Comment:
I guess these settings are specific to our environment and not needed anymore since we migrated Redmine away from the old environment?

Reply:
These settings were actually useless from the beginning, hence they were dropped. The `apiKey` suffices for authentication with redmine.
See https://github.com/adfinis-sygroup/timed-backend/pull/733